app/models/miq_alert_status_state.rb

```diff
@@ -0,0 +1,7 @@
+class MiqAlertStatusState < ApplicationRecord
```

Comment:
I'm having a hard time understanding why the domain was modeled in this way, which concerns me as we reflect this model in the API. What does it mean for a status to have a state?